demos/demo-react-ts/src/hooks/useCti.ts

```diff
@@ -90,25 +92,31 @@ class CallingExtensionsWrapper implements CallingExtensionsContract {
     this._iframeLocation = location;
   }
 
+  get usesCallingWindow() {
+    return this._usesCallingWindow;
+  }
+
+  set usesCallingWindow(usesCallingWindow: boolean) {
+    this._usesCallingWindow = usesCallingWindow;
+  }
+
   /** Do not send messages to HubSpot in the remote */
   get isFromRemote() {
-    return this._iframeLocation === "remote";
+    return this._usesCallingWindow && this._iframeLocation === "remote";
```

Comment:
if `usesCallingWindow` is true, we call the SDK events instead of doing an early return.

```
  userLoggedIn() {
    if (this.isFromRemoteOrWindow) {
      this.broadcastMessage({ type: thirdPartyToHostEvents.LOGGED_IN });
    }

    if (this.isFromRemote) {
      return;
    }

    return this._cti.userLoggedIn();
  }

```